uploader.html

```diff
@@ -147,29 +144,29 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
             <!-- selection section container -->
             <div id="js-selected-media-section">
               <script type="text/html">
```

Comment:
it's better practice to apply the template `id` to the `<script>` tag itself. And even better to put all templates together in their own section that's removed from the rest of the view (not nested in the `modal`). I know the original code wasn't following the best practices

Reply:
I like the idea of separating the functional from the stylistic, but I worry about having extra classes lying around.  

I also am not entirely sure I understand the "put all the templates in their own section" suggestion.  Would this entail creating another div id to replace the one that is now being assigned to the style tag?

Reply:
I was able to figure this out without adding additional classes.  Those changes can be found on the [media-uploader-sarah](https://github.com/dimagi/MediaUploader/tree/media-uploader-sarah) in the uploader.html file.